sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -99,6 +107,9 @@ def __init__(self, **kwargs: Any) -> None:
             # DistributedTracingPolicy(**kwargs),
             config.http_logging_policy or HttpLoggingPolicy(**kwargs)
         ]
+        if self._credential:
+            _add_credential_policy(policies, self._credential)
```

Comment:
We probably want to replace `config.authentication_policy` instead of appending a new one right?

Reply:
What do you think the behavior should be if they pass in a credential AND an auth policy? Might be good to keep both.

Reply:
I don't believe that is an exposed feature in our docs. Also, if the user does happen to pass in an auth policy and credential, that is undefined behavior since we don't know which one should take precedence. Ideally we want to only have one way of authenticating.

Reply:
Done.